service/service.hpp

```diff
@@ -0,0 +1,39 @@
+//
+// Created by milerius on 19/10/18.
+//
+
+#pragma once
+
+#include <filesystem>
+#include <memory>
+#include <uvw.hpp>
+
+namespace raven
```

Comment:
I'm not sure this is the best namespace name.
`libconfig` would maybe make more sense? It's a bit hard because the project has no real name yet

Reply:
Yeah that's why i put raven meanwhile this time, any inconvenience ?

Reply:
No, let's leave it at that for the moment.